README.md

```diff
@@ -295,19 +289,6 @@ helm delete --namespace=laa-apply-for-legalaid-uat <name-of-the-release>
 ```
 
 ## Dev: running locally
-
-### LAA Portal Authentication dev setup
-
-Add the following to .env.development
-```
-LAA_PORTAL_IDP_SSO_TARGET_URL=https://portal.tst.legalservices.gov.uk/oamfed/idp/samlv20
-LAA_PORTAL_IDP_CERT=< LAA_PORTAL_IDP_CERT (dev)>
-LAA_PORTAL_CERTIFICATE=< LAA_PORTAL_CERTIFICATE (dev)>
-LAA_PORTAL_SECRET_KEY=< LAA_PORTAL_SECRET_KEY(dev)>
-LAA_PORTAL_MOCK_SAML=false
-```
-** Note the above keys can be found in LastPass
-
```

Comment:
This is duplicated [here](https://github.com/ministryofjustice/laa-apply-for-legal-aid/blob/main/README.md?plain=1#L361-L376).